Fixed indentation on two images

lbugnion opened the pull request:
The code for two images was indented improperly, which caused the code to be displayed as Code instead of interpreted as a valid image.

azurecla commented:
Hi __@satonaoki__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@satonaoki - please recode the new table as markdown, not HTML. Thanks.

lbugnion committed:
Fixed indentation on two images

The code for two images was indented improperly, which caused the code to be displayed as Code instead of interpreted as a valid image.
